documentation/gnmi/design/sonic-dash.yang

```diff
@@ -0,0 +1,490 @@
+module sonic-dash {
+    namespace "http://github.com/Azure/sonic-dash";
+    prefix dash;
+    
```

Comment:
Hope you have a plan to add DASH YANG model for config-DB as well?

Reply:
We don't have plan to add DASH object to config_DB.